hudi-flink/src/main/java/org/apache/hudi/operator/InstantGenerateOperator.java

```diff
@@ -222,4 +236,59 @@ public void close() throws Exception {
       fs.close();
     }
   }
+
+  private boolean checkReceivedData(long checkpointId) throws InterruptedException, IOException {
+    int numberOfParallelSubtasks = getRuntimeContext().getNumberOfParallelSubtasks();
+    FileStatus[] fileStatuses = null;
+    Path generatePath = new Path(INSTANT_GENERATE_FOLDER_NAME);
+    int tryTimes = 1;
+    // waiting all subtask create generate file ready
+    while (true) {
+      Thread.sleep(500L);
+      fileStatuses = fs.listStatus(generatePath, new PathFilter() {
+        @Override
+        public boolean accept(Path pathname) {
+          return pathname.getName().contains(String.format("_%d_", checkpointId));
```

Comment:
You have defined the separator by `UNDERLINE`, it would be better to use that. wdyt?

Reply:
Can this suggestion be accepted?

Reply:
ok